packages/legacy/app/Gemfile

```
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby ">= 2.6.10"
-gem 'cocoapods', '~> 1.12'
-gem 'activesupport', '~> 7.0.8'
+gem 'cocoapods', '>= 1.13', '< 1.15'
+gem 'activesupport', '>= 6.1.7.5', '< 7.1.0'
```

```suggestion
gem 'cocoapods', '~> 1.14.3'
gem 'activesupport', '~> 7.0.8'
```
This could work, no? More precise, less potential variation in builds? Would still pick up patch version bumps but no minor or major ones.

---

I believe it could

---

@bryce-mcmath  resolved according to the versions you suggested